README.md

```diff
@@ -1 +1,47 @@
+[![lint](https://github.com/PermanentOrg/stela/actions/workflows/lint.yml/badge.svg)](https://github.com/PermanentOrg/stela/actions/workflows/lint.yml)
+[![unit tests](https://github.com/PermanentOrg/stela/actions/workflows/test.yml/badge.svg)](https://github.com/PermanentOrg/stela/actions/workflows/test.yml)
+[![codecov](https://codecov.io/gh/PermanentOrg/stela/branch/main/graph/badge.svg?token=4LYJGPGU57)](https://codecov.io/gh/PermanentOrg/stela)
```

Comment:
If you view the README as it exists on this branch, this badge won't populate properly because it's pointed at `main`, which hasn't uploaded any coverage reports to codecov yet. I have tested that it works properly if I point it at this branch, so I'm confident it will work on `main` once this is merged.